include/swift/ABI/Task.h

```diff
@@ -293,12 +663,19 @@ class AsyncTask : public HeapObject, public Job {
 
   FutureFragment *futureFragment() {
     assert(isFuture());
+
+    auto offset = reinterpret_cast<uintptr_t>(this); // TODO: char* instead?
```

Comment:
I'm not sure what style to follow -- other files do char* but this file seems to do uintptr_t for similar things.

WDYT / does it matter?